src/courses/mappers/10.md

```diff
@@ -0,0 +1,45 @@
+---
+order: 10
+next: 11.md
+title: Mapper Preparations - The Art of Mapper Making
+author: Charles Hu
+---
+
+## The Art of Mapper Making
+
+Mapper creation is an inherently tricky process as every situation demands a unique and tailored solution. As a result, there is no one objective best template or process to follow as field translations will always be designed on a case-by-case basis. 
+
+But don't fret! Mapper making, like many forms of development, is a skill you have to cultivate and improve at. So while there is no one-size-fits-all solution, you can still train yourself to produce mappers that are the best for your needs and uses.
+
+To help in improving your mapper making skills, the following section details a number of tips and best practices for mapper development that you should keep in mind while creating your mapper.
+
+## Mapper Making Best Practices
+
+### 1. Reach Out
+
+Don't be afraid to reach out to the SAF team for assistance with mapper development. Our developers can help you with everything from mapping decisions to mapper integration with OHDF Converters. 
+
+You can contact us through email (saf@groups.mitre.org) for inquiries and by creating a [GitHub issue on the Heimdall2 repository](https://github.com/mitre/heimdall2/issues/new/choose) for technical implementation issues.
+
+### 2. Reference Existing Solutions
+
+Try to browse and reference existing solutions in [OHDF Converters](https://github.com/mitre/heimdall2/tree/master/libs/hdf-converters) to glean some insight to how previous mappers were implemented. Through this, you might be able to find potential solutions to issues you encounter while making your mapper.
+
+### 3. Understand Your Security Tool and Data
+
+Understand how your security data is meant to be consumed in the original security tool's use case (i.e., don't just start development with the raw security data). Ask yourself the following questions:
+
+- How does the original security tool vendor group security data? Is it by type of concern? Where/what component was affected?
+- What associations does the tool make to common security references (e.g., NIST SP 800-53, CVE, CWE, CCE, OWASP, STIG IDs)?
+- What is the purpose of the security tool (e.g., vulnerability scanning, information visualization, etc.)?
+- How does the security tool organize its results (e.g., by requirement, by system, etc.)?
+
+These answers will help ensure that the intent of the data isn't skewed by the mapping process. It is important to try and "seek parity" with the original display style in the vendor's own tool. In other words, users must be able to recognize and navigate the security data in Heimdall just as easily as they can in the original native format.
+
+### 4. Use a Detailed Sample
+
+::: note APIs
+If your security tool provides an API instead of a security data export, contact the SAF team for further guidance in creating a specialized mapper that works with that API.
```

Comment:
I wouldn't say to immediately contact us.  Make it like "If your sec tool provides an api instead of a file export, you will need to explore the api to determine what information may need to be collected and collated for use within the mapper.  without there being an explicit results file from the vendor, you should have a thorough understanding of the security data that needs to be mapped onto hdf.  we would recommend getting in touch with the saf team blah blah".

Reply:
Addressed.